.readthedocs.yml

```
@@ -6,6 +6,9 @@ build:
     python: "mambaforge-4.10"
 
   jobs:
+    pre_build:
+      - mkdir -p $HOME/.ploomber/stats/
+      - echo "version_check_enabled: false" >> $HOME/.ploomber/stats/config.yaml
```

there's an issue when building the logs
this might fix it

```suggestion
    pre_build:
      - 'mkdir -p $HOME/.ploomber/stats/'
      - 'echo "version_check_enabled: false" >> $HOME/.ploomber/stats/config.yaml'
```

---

Yep stringifying fixed it, should I try the same with the previous `export ...` line?

---

no, export wont work